### ISSUE TITLE
Hearing API data not fresh/accurate

### ISSUE THREAD

**Issue by dylanirlbeck** [LibraryOfCongress/api.congress.gov#171]
Hi all. I have reason to believe that the `/hearing` endpoint is not returning fresh/accurate data for hearings. For example, this morning I `curl`ed the `/hearing/senate` endpoint to find the list of Senate hearings. [I documented the result](https://gist.github.com/dylanirlbeck/fc5d6cde90723263a5a000fd382c37e4) in a GitHub Gist.

Several upcoming and recent Senate hearings that I expected to be present in the dataset were not. Examples:
* [Hearings to examine ensuring Medicare beneficiary access, focusing on a path to telehealth permanency.](https://www.congress.gov/event/118th-congress/senate-event/335110?s=7&r=24)
* [Hearings to examine social media and the teen mental health crisis](https://www.congress.gov/event/118th-congress/senate-event/335066?s=4&r=6)

I'm going to switch to the `/committee-meeting endpoint` to see if I can get the data I need, but I would appreciate someone looking into the `/hearing` endpoint to see what's happening. Thanks!

**Comment by dylanirlbeck** [comment]
Is it possible that Senate hearings are simply not being coded correctly when they're entered into the system / scraped from `senate.gov`? All the Senate hearings are showing up as general committee meetings. This is in contrast to the House, which seems to better discriminate between hearings and meetings.
<img width="852" alt="Screenshot 2023-11-10 at 12 28 01 PM" src="https://github.com/LibraryOfCongress/api.congress.gov/assets/35497479/a9dc5657-74bc-4359-b758-56423d5e3caa">
<img width="852" alt="Screenshot 2023-11-10 at 12 28 08 PM" src="https://github.com/LibraryOfCongress/api.congress.gov/assets/35497479/7ac3410c-2275-4c02-b52c-98875c4ddff2">

**Comment by 104PL104** [comment]
Yes, you are correct that better structured data from the Senate is needed in order to differentiate between 'hearings' and 'business meetings', for example. We are in discussion with our Senate data partner about this requirement.

**Comment by dylanirlbeck** [comment]
@104PL104 @apreiter18 Any update on this? Thanks a bunch!

**Comment by 104PL104** [comment]
We do not yet have a timeline from our Senate data partner.
